Userland/Libraries/LibIPC/Decoder.h

```diff
@@ -27,6 +29,8 @@ inline ErrorOr<void> decode(Decoder&, T&)
     VERIFY_NOT_REACHED();
 }
 
+ErrorOr<void> decode(Decoder&, AK::Empty&);
```

Comment:
Does this need to be at the namespace level? Feels like it should be on the `Decoder` class like all other `decode` methods.

Reply:
Hmmm... I'm pretty sure I was thinking something when doing this but it has been too long. I'll try adding an overload.